docs/src/time_integration.md

```diff
@@ -0,0 +1,16 @@
+# Time integration methods
+
+Trixi is compatible with the [SciML ecosystem for ordinary differential equations](https://diffeq.sciml.ai/latest/).
+In particular, explicit Runge-Kutta from [OrdinaryDiffEq.jl](https://github.com/SciML/OrdinaryDiffEq.jl)
+are tested extensively.
+Interesting classes of time integration methods are
+- [Explicit low-storage Runge-Kutta time integration](https://diffeq.sciml.ai/latest/solvers/ode_solve/#Explicit-Strong-Stability-Preserving-Runge-Kutta-Methods-for-Hyperbolic-PDEs-(Conservation-Laws))
+- [Strong stability preserving methods](https://diffeq.sciml.ai/latest/solvers/ode_solve/#Low-Storage-Methods)
```

Comment:
```suggestion
- [Explicit low-storage Runge-Kutta time integration](https://diffeq.sciml.ai/latest/solvers/ode_solve/#Low-Storage-Methods)
- [Strong stability-preserving methods](https://diffeq.sciml.ai/latest/solvers/ode_solve/#Explicit-Strong-Stability-Preserving-Runge-Kutta-Methods-for-Hyperbolic-PDEs-(Conservation-Laws))
```
Missing `-` for SSP link text and flipped link URLs

Reply:
Haha, ok. I went to the link to find out how it's spelled and on the SciML docs they use the hyphen, but I am totally fine with it ;-)